pkg/query/querier.go

```diff
@@ -232,6 +238,11 @@ func newQuerierWithOpts(
 		skipChunks:              skipChunks,
 		shardInfo:               shardInfo,
 		seriesStatsReporter:     seriesStatsReporter,
+
+		enablePreAggregationLabelRewrite: opts.RewriteAggregationLabelTo != "",
+		aggregationStandardSuffixes:      []string{":aggr", ":count", ":sum", ":min", ":max", ":avg"},
```

Comment:
those are all constants, use them as what it is instead of adding variables to the construct